packages/stryker-mocha-runner/test/unit/MochaOptionsLoaderSpec.ts

```diff
@@ -3,7 +3,7 @@ import * as fs from 'fs';
 import { Config } from 'stryker-api/config';
 import MochaOptionsLoader from '../../src/MochaOptionsLoader';
 import { expect } from 'chai';
-import * as log4js from 'log4js';
+import * as log4js from 'stryker-api/logging';
```

Comment:
we call it log4js here and import the entire module

Reply:
Refactored the name, but the entire module is needed in order to mock away the `getLogger` method.